docs/index.rst

```diff
@@ -51,10 +54,103 @@ and high-level quantum programs. :ref:`Read more... <bloqs_library>`
 :ref:`API Reference <reference>`
 --------------------------------
 
-This section of the docs includes an API reference for all symbols in the library.
-:ref:`Go to Reference... <reference>`
+Each class, method, and function in the library is documented in the
+:ref:`API Reference<reference>`.
 
 .. toctree::
    :hidden:
 
    reference/index.rst
+
+
+Installation
+------------
+
+Basics
+~~~~~~
+
+Qualtran is a Python package. If you already have a working Python environment,
+
+.. code-block:: sh
+
+    pip install qualtran
+
+will install the library and its dependencies. Qualtran uses `Graphviz <https://graphviz.org/>`_
+to draw diagrams, which can't be installed using the Python package manager ``pip``. Follow
```

Comment:
Is it worth just reproducing these here (or adding a little script)

Reply:
Eh it's just apt-get nvm

Reply:
It's platform dependent, so I'd rather just point them to the actual docs.